pkg/service/publisher/publisher.go

```diff
@@ -15,22 +14,36 @@ import (
 	"github.com/multiformats/go-multiaddr"
 	"github.com/multiformats/go-multihash"
 	"github.com/storacha/go-capabilities/pkg/assert"
+	"github.com/storacha/go-capabilities/pkg/claim"
 	"github.com/storacha/go-metadata"
+	"github.com/storacha/go-ucanto/client"
 	"github.com/storacha/go-ucanto/core/delegation"
+	"github.com/storacha/go-ucanto/core/invocation"
+	"github.com/storacha/go-ucanto/core/receipt"
+	"github.com/storacha/go-ucanto/core/result"
+	"github.com/storacha/go-ucanto/did"
+	"github.com/storacha/go-ucanto/principal"
 	ipnipub "github.com/storacha/ipni-publisher/pkg/publisher"
 	"github.com/storacha/ipni-publisher/pkg/store"
+	"github.com/storacha/storage/pkg/capability"
 	"github.com/storacha/storage/pkg/service/publisher/advertisement"
 )
 
-const claimPattern = "{claim}"
-
 var log = logging.Logger("publisher")
 
+var (
+	AnnounceURL, _        = url.Parse("https://cid.contact/announce")
```

Comment:
I would make these CLI flags, with these as default values (also possible -- use the Value parameter). It's very confusing to pull them out of here and only use them in the CLI. unless they are used somewhere else?

Reply:
Ok I've moved these into `cmd/main.go`.